music21/musicxml/m21ToXml.py

```diff
@@ -4823,7 +4889,7 @@ def articulationToXmlTechnical(self, articulationMark):
             musicXMLTechnicalName = 'other-technical'
```

Comment:
We will need to return early to avoid getting the `<other-technical>`.

Reply:
Let's do it. Maybe the hammer-on doctest can also be moved to `noteToNotations` if not too much effort.

Reply:
Ah. I see. The doctest is useless now because we can't export unless we have a Stream object with the spanner in its spanner bundle. Okay. This is getting too complex for a doctest; we're just trying to see that the `<other-technical>` went away. Maybe you can add a couple lines to `testArticulationSpecialCases()`: add the HammerOn to the note, and then assert
```python
        self.assertIsNone(tree.find('.//other-technical'))
```
Thanks for bearing with me; once we get this we'll have CI passing and should be good.